### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by yosefalnajjarofficial**
- added the landing page component #17 
- added the ' / ' route and handled the landing page in it 
**Note:** It's mobile-first and it's not friendly on desktop view

### CODE REVIEW

**Comment by Fatmasiam** [comment]
Screenshot from 2019-09-30 01-43-05

:open_mouth::grimacing:

File: client/src/App.css [hunk 1]
```diff
@@ -1,6 +1,12 @@
-body {
+@import url('https://fonts.googleapis.com/css?family=Roboto&display=swap');
+
+* {
   padding: 0;
   margin: 0;
```
Comment (11janci) [inline client/src/App.css]: padding and margin are not inherited (ie. is 0 for every element by default, regardless of the parent's component), can be removed from here

File: client/src/components/pages/Landing/index.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    <div className="landing">
+      <div className="landing__logo">
+        <svg
+          width="188"
+          height="158"
+          viewBox="0 0 188 158"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path d={logo} fill="url(#paint0_linear)" />
```
Comment (11janci) [inline client/src/components/pages/Landing/index.js]: `logo` is undefined

Comment (yosefalnajjarofficial) [reply client/src/components/pages/Landing/index.js]: It is imported up in the same file from the svgPaths file and it displays the logo in the page

File: client/src/App.css (hunk 1 above)
Comment (yosefalnajjarofficial) [reply client/src/App.css]: It's for resetting the default margins and paddings which the browser adds to the elements

Comment (11janci) [reply client/src/App.css]: default margin and padding is zero

File: client/src/components/pages/Landing/index.js (hunk 2 above)
Comment (11janci) [reply client/src/components/pages/Landing/index.js]: I must be blind 🙄

**Pull request merged**